src/adapter/src/catalog.rs

```diff
@@ -2719,6 +2720,22 @@ impl Catalog {
                 }
             }
         }
+        // Operators aren't stored in the catalog, but we would like them in
+        // introspection views.
+        for (op, func) in OP_IMPLS.iter() {
```

Comment:
@benesch Are you ok with this approach?

Reply:
Absolutely. Moving operators into the catalog is wildly hard. See:

  * https://github.com/MaterializeInc/materialize/issues/9282
  * https://github.com/MaterializeInc/materialize/pull/6796